README.md

```diff
@@ -35,6 +35,18 @@ lane :release_docker_image do
   docker_push
 end
 
+lane :update_compose_deployment do
+  ensure_docker_machine_available(
+    machine_name: "docker-build-machine"
+  )
+  docker_compose(
+    command: "pull"
+  )
+  docker_compose(
+    command: "up"
+  )
+end
+
 ```
 
 This will start using a docker machine (creating it if it doesn't exist yet), login to docker hub, build and tag a new docker image and push that image to Docker Hub. 
```

Comment:
Can you move this up so that it's below the `:release_docker_image` lane, and then add a short explanation for the `:update_compose_deployment` one? Maybe something simple like `This will pull the compose environment and then start it` or something.